devices/philips.js

```diff
@@ -1636,9 +1673,16 @@ module.exports = [
         model: '4080248U9',
         vendor: 'Philips',
         description: 'Hue White and color ambiance Signe floor light',
-        toZigbee: [tzLocal.gradient_scene, ...hueExtend.light_onoff_brightness_colortemp_color({colorTempRange: [153, 500]}).toZigbee],
-        exposes: [exposes.enum('gradient_scene', ea.SET, Object.keys(gradientScenes)),
-            ...hueExtend.light_onoff_brightness_colortemp_color({colorTempRange: [153, 500]}).exposes],
+        toZigbee: [
+            tzLocal.gradient_scene,
+            tzLocal.philips_hue_multicolor({reverse: true}),
+            ...hueExtend.light_onoff_brightness_colortemp_color({colorTempRange: [153, 500]}).toZigbee,
+        ],
+        exposes: [
+            exposes.enum('gradient_scene', ea.SET, Object.keys(gradientScenes)),
+            exposes.text('philips_hue_multicolor', ea.SET),
```

Comment:
- I suggest to rename this to `colors`
- We can use the new list exposes instead: https://github.com/Koenkk/zigbee-herdsman-converters/blob/4c961732b32d34905b9b04738018c7b7b3b1384a/lib/exposes.js#L117

Reply:
Thanks for the quick review! I'll take a look and implement your suggested changes. :-)